### PR TITLE
fix: do not store output in test suite execution result

### PR DESCRIPTION
**Pull request by nicufk**
## Pull request description 
Closes https://github.com/kubeshop/testkube/issues/4410

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go [hunk 1]
```diff
@@ -243,3 +243,12 @@ func (e *TestSuiteExecution) EscapeDots() *TestSuiteExecution {
 func (e *TestSuiteExecution) UnscapeDots() *TestSuiteExecution {
 	return e.convertDots(utils.UnescapeDots)
 }
+
```
Comment (vsukhin) [inline pkg/api/v1/testkube/model_test_suite_execution_extended.go:246]: in fact, it's for TestSuite v2, for TestSuite v3 we need to clean BatchSteps

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go [hunk 2]
```diff
@@ -243,3 +243,12 @@ func (e *TestSuiteExecution) EscapeDots() *TestSuiteExecution {
...
+func (e *TestSuiteExecution) CleanStepsOutput() *TestSuiteExecution {
+	for i := range e.StepResults {
+		if e.StepResults[i].Execution != nil && e.StepResults[i].Execution.ExecutionResult != nil {
+			e.StepResults[i].Execution.ExecutionResult.Output = ""
```
Comment (vsukhin) [inline pkg/api/v1/testkube/model_test_suite_execution_extended.go:250]: might be we need to give a reference to test execution log?

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go [hunk 3]
```diff
@@ -248,6 +248,9 @@ func (e *TestSuiteExecution) CleanStepsOutput() *TestSuiteExecution {
 	for i := range e.StepResults {
 		if e.StepResults[i].Execution != nil && e.StepResults[i].Execution.ExecutionResult != nil {
 			e.StepResults[i].Execution.ExecutionResult.Output = ""
+			for j := range e.ExecuteStepResults[i].Execute {
+				e.ExecuteStepResults[i].Execute[j].Execution.ExecutionResult.Output = ""
```
Comment (vsukhin) [inline pkg/api/v1/testkube/model_test_suite_execution_extended.go:252]: should we check it for nil values?

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go (hunk 1 above)
Comment (nicufk) [reply pkg/api/v1/testkube/model_test_suite_execution_extended.go:246]: Done

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go (hunk 2 above)
Comment (nicufk) [reply pkg/api/v1/testkube/model_test_suite_execution_extended.go:250]: Added api spec note

File: pkg/api/v1/testkube/model_test_suite_execution_extended.go (hunk 3 above)
Comment (nicufk) [reply pkg/api/v1/testkube/model_test_suite_execution_extended.go]: Done

**Review by vsukhin — APPROVED** [review]
good to go

**Pull request merged**